### ISSUE TITLE
Link cards does not inherit Label and Customfields

### ISSUE THREAD

**Issue by marcungeschikts** [wekan/wekan#3748]
## Issue
**Problem description:**
When creating a link card, labels and custom fields are not inherited even if they are defined in both boards.

See video attached:
https://user-images.githubusercontent.com/74183327/115690361-4764b380-a35d-11eb-9f85-608cb145c1b4.mp4

**Server Setup Information:**
Did you test in newest Wekan?: YES
Wekan version: last version
Deployment Method :snap
Webbrowser version :Chrome Version 89.0.4389.90 (Build officiel) (64 bits)

**Comment by ryanMushy** [comment]
This happens to an extent when you copy a template card into a board.
If the new board has no labels and fields, the template card gets created without those fields or labels

**Comment by marcungeschikts** [comment]
Hi Ryan,
The card in the video attached was not a template card and the label and custom fields was both created identically in both boards.

**Comment by ryanMushy** [comment]
indeed, my message was an attempt to group similar behaviour together. It is my opinion that the ability to "copy" a card into the local board has several deficiencies that could be addressed all at the same time.

It would by my assessment that when copying/linking/templating a card, all needed entities for that card must exist on the board (if it does not exist, it must be created programmatically) and then those entities should be attached to the card. 

This flow is not happening for all types of "copy" actions that exist for a card, and moreover, each type of copy has a different outcome
